#!/usr/bin/env python
# field of view configuration

PACKAGE="cob_table_object_cluster"
import roslib; roslib.load_manifest(PACKAGE)

from math import pi

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("height_max", double_t, 3, "upper boundary of cluster search area", -0.03, -0.5, 0)
gen.add("height_min", double_t, 4, "lower boundary of cluster search area", -0.5, -1.5, 0)
gen.add("min_cluster_size", double_t, 5, "minimum number of points for a valid cluster", 10, 0, 1000)
gen.add("cluster_tolerance", double_t, 6, "threshold for separation of two clusters", 0.04, 0, 0.2)
gen.add("save", bool_t, 2, "Save results", False)
gen.add("file_path", str_t, 1, "File path to store results", "")

exit(gen.generate(PACKAGE, "dynamic_reconfigure_node", "table_object_cluster_nodelet"))
